main/ertp/ERTP-Guide/Overview.md

```diff
@@ -0,0 +1,276 @@
+them all on Captain Kirk to win his gladitorial match on Triskelion (see the[ Wikipedia entry](https://en.wikipedia.org/wiki/The_Gamesters_of_Triskelion)).
+
+**tyg todo: Needs beefing up on how allegedNames are used in contracts
+  vs. getting and using the real objects**
+
+## Creating and sending a fungible asset
+
+In ERTP, digital assets are created by a [`mint`](). You must have
+access to a `mint` object to create new assets of the `mint` object's
+associated `brand`. 
+
+For instance, let's say we want to create a new community
+currency called 'BaytownBucks'. You start by getting access to a `mint`
+object whose associated brand is 'BaytownBucks'.
+```js
+import { makeMint } from '@agoric/ertp';
+const baytownBucksMint = makeMint('BaytownBucks');
+```
+
+Now, use the `mint` to create 1000 new BaytownBucks.
+
+```js
+const { issuer, mint } = produceIssuer('BaytownBucks');
+const BB1000 = amountMath.make(1000);
+const newPayment = mint.mintPayment(BB1000);
+const BBpurse = issuer.makeEmptyPurse();
+const BB100Amount = BBpurse.deposit(newPayment);
+```
+
+The act of minting created 1000 BaytownBucks and stored them together in a
+`payment`. We then deposited the `payment` into a `purse` for BaytownBucks.
+
+Now, let's distribute the BaytownBucks to others. To send assets
+in ERTP, we withdraw a `payment` from a `purse`.
+
+```js
+const paymentForAlice = purse.withdraw(10, `alice's community money`);
+```
+
+Like our `purse`, this `payment` contains BaytownBucks, but unlike purses,
+payments are used to represent tokens in transit. A `payment` can be
+sent to someone else, a `purse` should never be sent to someone else.
+
+Now let's send the `payment` to Alice as a message:
+
+```js
+alice.receivePayment(paymentForAlice);
+```
+
+Here `alice` is an object reference, and we can call her `receivePayment`method
+to ask her to receive this payment. Alice's methods are entirely up to her, 
+and are not part of ERTP.
+
+
+### Receiving the asset
+
+Before she can receive a `payment` in BaytownBucks, Alice needs to 
+create an empty `purse` for BaytownBucks and otherwise get ready to
+receive payments. 
+For this to happen, Alice needs access to `baytownBucksIssuer`.
+However, she does not need access to `baytownBucksMint`.
+If she had access to the `mint`, she could create BaytownBucks herself
+by calling `baytownBucksMint.mint`. 
+
+```js
+const myBaytownBucksPurse = baytownBucksIssuer.makeEmptyPurse()
+
+myBaytownBucksPurse.getAmount(); // Empty, so the value is 0
+
+const alice = {
+    receivePayment(allegedBaytownBucksPayment){
```

Comment:
see comment above about the depositFacet and board to show how it should actually be done.